itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/AbstractNewQuarkusProjectTest.java

```diff
@@ -144,10 +155,52 @@ public static void checkProblemsView() {
 
 	}
 
+	public void runProjectWithDebug(String projectName) {
```

Comment:
I would keep this function in RunProjectWithDebugTest, because it is quite specific and I do not see any other usecase for this.

Reply:
fixed.